table/two_level_iterator.cc

```diff
@@ -24,7 +24,11 @@ class TwoLevelIterator : public InternalIterator {
   explicit TwoLevelIterator(TwoLevelIteratorState* state,
                             InternalIterator* first_level_iter);
 
-  virtual ~TwoLevelIterator() { delete state_; }
+  virtual ~TwoLevelIterator() {
+    first_level_iter_.DeleteIter(false);
```

Comment:
what is the variable that is assigned to false? and why false?

Reply:
it's `is_arena_mode` that dictates whether to call `delete` or `~InternalIterator()`
I think by default it was false for both first and second level iter in the original code https://github.com/facebook/rocksdb/pull/3406/files#diff-5fed9b271c2846509d9ad683c2624f26

Reply:
- Can you put a comment beside the false value? something like `/* is_arena_mode */`
- What if later someone uses TwoLevelIterator and passes a first_level_iterator is created using arena? Can we add some comments to NewTwoLevelIterator and NewSecondaryIterator to indicate the contract that the object should not have been created using arena.